src/extension/background-script/actions/allowances/update.ts

```diff
@@ -0,0 +1,25 @@
+import db from "~/extension/background-script/db";
+import type { MessageAllowanceUpdate } from "~/types";
+
+const updateAllowance = async (message: MessageAllowanceUpdate) => {
+  const id = message.args.id;
+
+  const update: {
+    totalBudget?: number;
+    remainingBudget?: number;
+    enabled?: boolean;
+  } = {};
+  if (Object.prototype.hasOwnProperty.call(message.args, "totalBudget")) {
+    update.totalBudget = message.args.totalBudget;
+    update.remainingBudget = message.args.totalBudget;
+  }
+  if (Object.prototype.hasOwnProperty.call(message.args, "enabled")) {
```

Comment:
can you check if we use this? doe we call updateAllowance somewhere with setting the enabled flag? 

(I don't think so, even though somebody requested that in telegram just yesterday)

Reply:
related: https://github.com/getAlby/lightning-browser-extension/issues/1142